common/db.go

```diff
@@ -94,6 +101,13 @@ var schema = `
 	);
 
 	CREATE INDEX IF NOT EXISTS idx_winningtickets_sessionid ON winningTickets(sessionID);
+
+	CREATE TABLE IF NOT EXISTS blockheaders (
+		number int64 PRIMARY KEY,
+		parent STRING,
+		hash STRING,
```

Comment:
I think the primary key should actually be `hash` since each block header is uniquely identified by its hash (there could be 2 block headers with the same `number` but different hashes).

Reply:
Fixed in ced063e